.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+     - master
+
+env:
+  DISPLAY: :100.0
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12, 14, 16]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'yarn'
+      - uses: browser-actions/setup-firefox@latest
+        with:
+          firefox-version: "latest"
```

Comment:
Can you use single or double spaces consistently in this file?

Reply:
@jeffcarp 
Thanks for your comment! I fixed it.
https://github.com/jsdom/abab/pull/77/commits/a7883fc927817a9c691232515160f5b08b595ca7

Reply:
Thanks!